docs/sources/administration/correlations/use-variables-and-transformations/index.md

```diff
@@ -3,7 +3,7 @@ labels:
   products:
     - enterprise
     - oss
-title: Use variables and transformations in a correlation
+title: 'Example: Use variables and transformations in a correlation'
```

Comment:
I don't think this change should be made as it changes the information architecture of the section.

I believe this is a "task topic" and should conform to the advice in https://grafana.com/docs/writers-toolkit/structure/topic-types/task/

```suggestion
title: Use variables and transformations in a correlation
```